tools/cldr-code/src/main/java/org/unicode/cldr/util/PatternPlaceholders.java

```diff
@@ -106,6 +110,12 @@ public PlaceholderData transform(String source) {
                         } else {
                             example = "";
                         }
+                        // TODO Some normalization of personName namePattern placeholder ids.
+                        // Hack for now, later do something maybe using PersonNameFormatter.ModifiedField
+                        id = id.replace("-allCaps", "");
+                        id = id.replace("-initialCap", "");
+                        id = id.replace("-initial", "");
+                        id = id.replace("-monogram", "");
```

Comment:
Should include all the modifiers?

Reply:
@macchiati No because I have explicit placeholders for {given-informal} {surname-prefix} {surname-core}, per the summary above. I am only ignoring (stripping) the modifiers that can be used with multiple fields.